code/game/objects/items/pillow.dm

```diff
@@ -0,0 +1,160 @@
+	w_class = WEIGHT_CLASS_NORMAL
+	damtype = STAMINA
+	var/static/tag_desc = "This one seems to have its tag removed."
+	var/obj/item/clothing/neck/pillow_tag/pillow_trophy
+	///whoever last use this pillow
+	var/last_fighter
+	///for selecting the various sprite variation, defaults to the blank white pillow
+	var/variation = 1
+
+
+/obj/item/pillow/Initialize(mapload)
+	. = ..()
+	if(!pillow_trophy)
+		pillow_trophy = new(src)
+	AddComponent(/datum/component/two_handed, \
+		force_unwielded = 5, \
+		force_wielded = 10, \
+	)
+
+/obj/item/pillow/Destroy(force)
+	. = ..()
+	QDEL_NULL(pillow_trophy)
+
+/obj/item/pillow/attack(mob/living/carbon/target_mob, mob/living/user, params)
+	. = ..()
+	if(!iscarbon(target_mob))
+		return
+	if(HAS_TRAIT(src, TRAIT_WIELDED))
+		user.apply_damage(5, STAMINA) // when hitting with such force we should prolly be getting tired too
+	last_fighter = user
+	playsound(user, 'sound/items/pillow_hit.ogg', 80) //the basic 50 vol is barely audible
+
+/obj/item/pillow/attack_secondary(mob/living/carbon/victim, mob/living/user, params)
+	. = ..()
+	if(!isliving(victim) || isdead(victim))
+		return
+	if(victim.wear_mask || !user.get_bodypart(BODY_ZONE_HEAD))
+		return
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		to_chat(user, span_notice("You can't bring yourself to harm [victim]"))
+		return
+	if(victim.body_position || user.grab_state >= GRAB_AGGRESSIVE)
+		user.visible_message("[user] starts to smother [victim]", span_notice("You begin smothering [victim]"), vision_distance = COMBAT_MESSAGE_RANGE)
+		smothering(user, victim)
+
+/obj/item/pillow/proc/smothering(mob/living/carbon/user, mob/living/carbon/victim)
+	while(victim)
+		if(victim.body_position == FALSE && user.grab_state <= GRAB_NECK)
+			break
+		if(!do_after(user, 1 SECONDS, victim))
+			break
+		victim.losebreath = 1
+	victim.losebreath = 0
+	victim.visible_message("[victim] manages to escape being smothered!", span_notice("You break free!"), vision_distance = COMBAT_MESSAGE_RANGE)
+
+/obj/item/pillow/examine(mob/user)
+	. = ..()
+	. += span_notice("<i>There's more information below, you can look again to take a closer look...</i>")
+
+/obj/item/pillow/examine_more(mob/user)
+	. = ..()
+	. += span_notice("Alt-click to remove the tag!")
+
+/obj/item/pillow/AltClick(mob/user)
+	. = ..()
+	if(!pillow_trophy)
+		balloon_alert(user, "no tag!")
+		return
+	balloon_alert(user, span_notice("removing tag..."))
+	if(!do_after(user, 2 SECONDS, src))
+		return
+	if(last_fighter)
+		pillow_trophy.desc = "a pillow tag taken from [last_fighter] after a gruesome pillow fight."
+	user.put_in_hands(pillow_trophy)
+	pillow_trophy = null
+	balloon_alert(user, span_notice("tag removed"))
+	playsound(user,'sound/items/poster_ripped.ogg', 50)
+	update_appearance()
+	
+
+/obj/item/pillow/update_appearance(updates)
+	. = ..()
+	if(!pillow_trophy)
+		desc = "A soft and fluffy pillow. You can smack someone with this! [tag_desc]"
+		icon_state = "pillow_[variation]"
```

Comment:
Should this be reversible if a pillow trophy is added back on?

Reply:
I dont plan to allow it to be glued on so no

Reply:
ok nvm its done, pillow tag can now be reattached